servicenow/sample-templates/blueprint.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: xl/v1
+kind: Blueprint
+metadata:
+  projectName: ServiceNow Demo Application
+  description: |
+    The blueprint supplies two yaml files with sample templates for usage with the XL Release ServiceNow plugin and optionnaly the ServiceNow application 'XL Release Integration' from teh appstore.
+  author: XebiaLabs
+  version: 1.0
+  instructions: Please read the generated file "xebialabs/USAGE.md" for further usage instructions.
+spec:
+  parameters:
+  - name: ServerName
+    type: Input
+    description: Provide a name for the ServiceNow server?
+  - name: ServiceNowURL
```

Comment:
is it possible to provide a ServiceNow demo container image?

Reply:
Not a recent version that is supported by ServicNow. I have zero docker experenice, so can't say if creating one is even an option.

Reply:
How can we test this blueprint? Can you provide us some guidelines for someone that has zero ServiceNow experience?